plugins/woocommerce-admin/client/customize-store/assembler-hub/layout.tsx

```diff
@@ -52,6 +52,7 @@ import { AiOfflineModal } from '~/customize-store/assembler-hub/onboarding-tour/
 import { useQuery } from '@woocommerce/navigation';
 import { FlowType } from '../types';
 import { isOfflineAIFlow } from '../guards';
+import { isWooExpress } from '~/utils/is-woo-express';
```

Comment:
For now, we can keep this conditional as-is. Still, it is worth mentioning that as a next step for this project, we are abstracting all references to WooExpress and AI from the core of WooCommerce (to be worked on as part of separate/upcoming PRs).